src/dcs/adapters/outbound/http/exceptions.py

```diff
@@ -24,7 +24,8 @@ class BadResponseCodeError(KnownError):
 
     def __init__(self, *, url: str, response_code: int):
         self.response_code = response_code
-        message = f"The request to {url} failed with response code {response_code}"
+        message = f"The request to {
+            url} failed with response code {response_code}"
```

Comment:
Did ruff do that?

Reply:
There was some weird behaviour with formatting only resolved by rebuilding the container... missed fixing this one